feat(genapi): update model names in tutorial (#6580)
scaleway/docs-content: pages/generative-apis/reference-content/integrate-with-cursor.mdx

---
title: Integrating Generative APIs with Cursor

> description: Learn how to integrate Scaleway Generative APIs with Cursor AI code editor for enhanced coding assistance using models like qwen3.6-35b-a3b. Includes visual configuration guide.

Author: ```suggestion
description: Learn how to integrate Scaleway Generative APIs with Cursor AI code editor for enhanced coding assistance using models like qwen3.5-397b-a17b. Includes visual configuration guide.
```

Cursor is an AI-powered code editor that helps developers write and edit code more efficiently. It can read your existing codebase, help you navigate it, and generate code based on your natural language descriptions. Built on top of Visual Studio Code, it allows you to integrate language models directly into your coding workflows.

> This guide will help you integrate Scaleway Generative APIs into Cursor. This lets you use models such as `qwen3.5-397b-a17b` for code assistance and generation.

Reviewer: ```suggestion
This guide shows you how to integrate Scaleway Generative APIs into Cursor, enabling you to use models such as `qwen3.5-397b-a17b` for code assistance and generation.
```

    <Message type="tip">
    If you are using a Project other than your [default Project](/organizations-and-projects/concepts/#default-project), modify the base URL accordingly: `https://api.scaleway.ai/{project_id}/v1`
    </Message>

> 8. In the **Models** search bar at the top of the page, paste the name of the model you want to use, for example, `qwen3.6-35b-a3b`.

Author: ```suggestion
8. In the **Models** search bar at the top of the page, paste the name of the model you want to use, for example, `qwen3.5-397b-a17b`.
```

We recommended using the following models with Cursor:

> - `qwen3.6-35b-a3b` as the default choice for most tasks, especially autocompletion.

Reviewer: ```suggestion
- `qwen3.6-35b-a3b` as the default choice for most tasks, especially autocompletion
```

> - `qwen3.5-397b-a17b` for agentic tasks.

Reviewer: ```suggestion
- `qwen3.5-397b-a17b` for agentic tasks
```

> - `qwen3-embedding-8b` for embeddings.

Reviewer: ```suggestion
- `qwen3-embedding-8b` for embeddings
```
